README.md

```diff
@@ -16,12 +16,48 @@ Communication
 The easiest way to [communicate with us is by opening a new issue](https://github.com/savoirfairelinux/ringme.js/issues/new).
 Make sure you add proper labels to your issue (e.g. question, issue, enhancement, …)
 
+Usage
+-----
+
+### Why you don't need it
+
+This project does nothing except proposing a default look & feel for a
+`Ring Me` button that you can add to any web page _easily_.
+
+You could simply choose to not use this Javascript project and add a
+link on your page like so:
+
+`<a href="ring:my-ring-ID">Call me with Ring, a communication tool that
+respects your freedoms and privacy</a>`
+
+as you would add a link on your page to let your visitors email you:
+
+`<a href="mailto:my-email@example.com">Email me!</a>`
+
+The magic happens through the Ring client that configures your computer
+to handle the `ring:` scheme.
+
+### Why you need it
+
+Since Ring clients are not spread as much as email clients (yet), this
+little project helps in identifying if `ring:` scheme can be handled by
+the visitor of a web page.
```

Comment:
Since Ring clients are not as widely used as email clients (yet), this little project helps by identifying if the `ring:` scheme can be handled by a visitor's computer.

Reply:
Fixed by 40e83fb